frontend/package.json

```diff
@@ -21,7 +23,7 @@
     "eject": "react-scripts eject"
   },
   "engines": {
-    "node": "14.4.0"
+    "node": "10.x.x"
```

Comment:
why downgrade?

Reply:
For deployment purposes, the appengine requires the version to be of nodejs10. Locally, either version works.